manifests/o/Oracle/MySQLNotifier/1.1.8/Oracle.MySQLNotifier.locale.en-US.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Oracle.MySQLNotifier
+PackageVersion: 1.1.8
+PackageLocale: en-US
+Publisher: Oracle
+PublisherUrl: https://oracle.com
+PrivacyUrl: http://www.oracle.com/us/legal/privacy/index.htm
+PackageName: MySQL Notifier 1.1.8
```

Comment:
```suggestion
PackageName: MySQL Notifier
```

Reply:
@ItzLevvie I thought this had to match the ARP entry?

Reply:
Yes, but for now with the version removed as we're waiting for the 1.1.0+ schema for that so there can be a name for WinGet and another name for ARP matching.

It also screws up the `winget list` command for those who have an older version installed. For example: shows that the user has  `MySQL Notifier 1.1.8` installed on their machine but they actually don't. This most likely won't happen for most MSI installers because ProductCode is being used for matching, except for those installers that have the same ProductCode for every new version, but will happen to EXE.

Reply:
Thanks for the explanation!